src/python-common/ceph/deployment/service_spec.py

```diff
@@ -45,6 +45,13 @@ def inner(cls, *args, **kwargs):
     return inner
 
 
+def unwrap_ipv6(address):
+    # type: (str) -> str
+    if address.startswith('[') and address.endswith(']'):
+        return address[1:-1]
+    return address
+
+
```

Comment:
I don't like having to keep defining the same function, but cephadm should be self containing, orch grew them, and then we needed one in ceph.deployment as all these files seem to also seem to be disconnected from ceph and the orch.

Happy to put something in like:
`from orchestrator import unwrap_ipv6` 

But doesn't seem the right thing to do.

Reply:
the other way around works! you can import python-common from mgr/